python/sauron-core/tests/test_classifier.py

```
@@ -0,0 +1,157 @@
+import pytest
+import numpy as np
+from core.analysis.classifier import AIImageClassifier
+
+def test_classifier_initialization():
+    """Test classifier initialization"""
+    classifier = AIImageClassifier()
+    assert classifier.classifier is not None
+    assert classifier.scaler is not None
+    assert len(classifier.feature_columns) > 0
+
+def test_prepare_features():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_prepare_features`

Add return type annotation: `None`

(ANN201)